src/streams.c

```diff
@@ -1232,6 +1232,21 @@ static Obj FuncIsDirectoryPathString(Obj self, Obj filename)
     return res == -1 ? False : True;
 }
 
+/****************************************************************************
+**
+*F  FuncIsWritablePath( <self>, <name> ) . . . .  is file <name> a directory
```

Comment:
that description is wrong, and the variable `name` does not match `filename` below